src/utils/normalizeModuleFederationOptions.ts

```diff
@@ -289,6 +293,7 @@ export type ModuleFederationOptions = {
             singleton?: boolean;
             requiredVersion?: string;
             strictVersion?: boolean;
+            import?: boolean;
```

Comment:
In the [mfe docs](https://module-federation.io/configure/shared) is a string or boolean 🤔

Reply:
updated to use the shared plugin interface type `sharePlugin.SharedConfig['import']`